.storybook/custom-theme.css

```diff
@@ -82,4 +82,10 @@ nav.container.sidebar-container.css-t68aeq {
     top: -107px !important;
     left: -10px !important;
     position: relative !important;
+}
+
+.css-1rb1jn6 {
```

Comment:
@poojagurav21 why this class is added please confirm on this

Reply:
Reduced the size of Raaghu logo. Confirm with Kaushik.

Reply:
@poojagurav21 , please resolved conflicts